docs/core/tools/dotnet-nuget-verify.md

```diff
@@ -0,0 +1,81 @@
+---
+title: dotnet nuget verify command
+description: The dotnet nuget verify command verifies a signed package.
+author: kartheekp-ms
+ms.date: 10/08/2020
+---
+# dotnet nuget verify
+
+**This article applies to:** ✔️ .NET 5.0.100-rc.2.x SDK and later versions
+
+## Name
+
+`dotnet nuget verify` - Verifies a signed NuGet package.
+
+## Synopsis
+
+```dotnetcli
+dotnet nuget verify [<package-path(s)>]
+    [--all]
+    [--certificate-fingerprint <FINGERPRINT>]
+    [-v|--verbosity <LEVEL>]
+
+dotnet nuget verify -h|--help
+```
+
+## Description
+
+The `dotnet nuget verify` command verifies a signed NuGet package.
+
+## Arguments
+
+- **`package-path(s)`**
+
+  Specifies the file path to the package(s) to be verified. Multiple position arguments can be passed in to verify multiple packages.
+
+## Options
+
+- **`--all`**
+
+  Specifies that all verifications possible should be performed on the package(s). By default, only `signatures` are verified.
+
+> [!NOTE]
+> This command currently supports only `signature` verification. Additional quality checks shall be added in the future.
```

Comment:
Suggest we delete promise of future functionality -- See Future product plans or promises in https://review.docs.microsoft.com/en-us/help/contribute/contribute-get-started-channel-guidance?branch=master

```suggestion
> This command currently supports only `signature` verification.
```